src/frameworks/flutter/utils.ts

```diff
@@ -8,3 +10,27 @@ export function assertFlutterCliExists() {
       "Flutter CLI not found, follow the instructions here https://docs.flutter.dev/get-started/install before trying again.",
     );
 }
+
+export async function getTreeShakeFlag(): Promise<string> {
```

Comment:
I think we can make it generic like `getAdditionalBuildArgs` and make it return an array of strings, then we can accommodate more use cases in the future if needed